_overviews/tutorials/scala-for-java-programmers.md

```diff
@@ -6,19 +6,79 @@ partof: scala-for-java-programmers
+    link: /scala3/book/scala-for-java-devs.html
+  - title: Online Courses
+    description: MOOCs to learn Scala, for beginners and experienced programmers.
+    icon: "fa fa-cloud"
+    link: /scala3/book/scala-for-java-devs.html
 ---
 
-By Michel Schinz and Philipp Haller
+If you are coming to Scala with some Java experience already, this page should give a good overview of
+the differences, and what to expect when you begin programming with Scala. For best results we suggest
+to either set up a Scala toolchain on your computer, or try compiling Scala snippets in the browser with Scastie:
+
+{% include inner-documentation-sections.html links=page.get_started_resources %}
+
+## At a Glance: Why Scala?
+
+**Java without Semicolons:** There's a saying that Scala is Java without semicolons.
+There is a lot of a truth to this statement: Scala simplifies much of the noise and boilerplate of Java,
+while building upon the same foundation, sharing the same underlying types and runtime.
+
+**Seamless Interop:** Scala can use any Java library out of the box; including the Java standard library!
+And pretty much any Java program will work the same in Scala, just by converting the syntax.
+
+**A Scalable Language:** the name Scala comes from Scalable Language. Scala scales not only with hardware
+resources and load requirements, but also with the level of programmer's skill. If you choose, Scala
+rewards you with expressive additional features, which when compared to Java, boost developer productivity and
+readability of code.
+
+**It Grows with You:** Learning these extras are optional steps to approach at your own pace.
+The most fun and effective way to learn, in our opinion, is to ensure you are productive first with what knowledge
+you have from Java. And then, learn one thing at a time following the [Scala Book][scala-book]. Pick the learning pace convenient for you and ensure whatever you are learning is fun.
+
+**TL;DR:** You can start writing Scala as if it were Java with new syntax, then explore from there as you see fit.
+
+## Next Steps
+
+### Compare Java and Scala
+The remainder of this tutorial expands upon some of the key differences between Java and Scala,
+with further explanations. **If you only want a quick reference** between the two, read
+*Scala for Java Developers*, it comes
+with many snippets which you can try out in your chosen Scala setup:
 
-## Introduction
+{% include inner-documentation-sections.html links=page.java_resources %}
 
-This document gives a quick introduction to the Scala language and
-compiler. It is intended for people who already have some programming
-experience and want an overview of what they can do with Scala. A
-basic knowledge of object-oriented programming, especially in Java, is
-assumed.
+### Explore Further
 
-## A First Example
+When you finish these guides, we recommend to continue your Scala journey by reading the
+*Scala Book* or following a number of *online MOOCs*.
```

Comment:
These should probably be links to the corresponding pages (Scala book and online courses)

Reply:
these are immediately followed by the relevant links - so I do not think it makes sense:

<img width="839" alt="Screenshot 2022-08-23 at 16 05 07" src="https://user-images.githubusercontent.com/13436592/186179174-94c7fa00-b12a-4b61-916b-ce05576e8f1a.png">

Reply:
perhaps it would not be confusing to link twice